app/jobs/catalog_job.rb

```diff
@@ -0,0 +1,43 @@
+# CatalogJob is a job that synchronizes the contents of the database with the
+# contents of the Registry. This is done by using the Catalog API.
+class CatalogJob < ActiveJob::Base
+  # This method will be called on each tic of the cron. It basically gets the
+  # contents from the registry and calls `update_registry!`. Any error is
+  # logged.
+  def perform
+    registry = Registry.first
```

Comment:
Can you add a `# TODO: change the code once allow more registries to be configured at the same time`. We already did in some parts of the code if I'm right, it should make everything easier later on if we decide to implement that feature.